config.js

```diff
@@ -339,9 +339,20 @@ var config = {
     //      "libs/analytics-ga.js", // google-analytics
     //      "https://example.com/my-custom-analytics.js"
     // ],
+    analytics: {
+        // The Google Analytics Tracking ID
+        // The Google Analytics Tracking ID:
+        // googleAnalyticsTrackingId: 'your-tracking-id-UA-123456-1'
```

Comment:
There is some copy / paste error here.

Reply:
IMHO, having `googleAnalyticsTrackingId` inside an `analytics` section sounds redundant. How about `googleTrackingId` ?

Reply:
I agree although the analytics platform is called "Google Analytics"....